src/main/java/com/amplitude/Event.java

```diff
@@ -10,34 +10,104 @@
 public class Event {
 
     public static final String TAG = "com.amplitude.Event"; //AmplitudeClient.class.getName();
```

Comment:
why do you need this TAG

Reply:
We don't need the tag for now. Later on we'll add a few dedicated tagger (debug printer) so that people's apps won't be overridden with errors (for example, HTTP 400 response from Amplitude endpoint).

Reply:
Then it should be Event.class.getName().